### PR TITLE
linking all the pages together for student user journey

### PR DESCRIPTION
**Pull request by nairdesigns**
only the first link/ exercise  works:  it comes form this page 

image

to this page

image

there is something wrong, that I can't figure out with the rest of the links. I think it has to do something with the seeds and how the other Exercises don't have any instance of music.

image

### CODE REVIEW

File: app/views/exercises/_results_card.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="card-result modal fade mx-5 p-5" id="commentModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <h1 class="text-center pt-5 mb-4">Congratulations!</h1>
```
Comment (markiecyt) [inline app/views/exercises/_results_card.html.erb:2]: If you  put `modal-dialog` class here on the h1 it should start to work but it is missing some other things like the dismiss button and other bootstrap divs/classes

**Comment by nairdesigns** [comment]
modal is working now!

image

**Comment by nairdesigns** [comment]
[submit button temporarily leads to exerecies page] for our user journey illustration

**Comment by nairdesigns** [comment]
Please do!

**Pull request merged**